api/filters/namespace/namespace.go

```diff
@@ -19,6 +19,9 @@ type Filter struct {
 	// FsSlice contains the FieldSpecs to locate the namespace field
 	FsSlice types.FsSlice `json:"fieldSpecs,omitempty" yaml:"fieldSpecs,omitempty"`
 
+	// SkipExisting means only blank namespace fields will be set
+	SkipExisting bool `json:"skipExisting" yaml:"skipExisting"`
```

Comment:
Naming suggestions welcome! ( I want it to have negative polarity though, so false when missing give the current default behaviour)

Reply:
An alternative idea I had was `EmptyOnly` or `BlankOnly - it doesn't reverse the polarity but reverses the wording to focus on the fact that this will only set empty/blank namespace fields. I like this slightly better because it focuses on what does get changed, rather than what doesn't.

I'll defer to what you choose though, I think either way is fine.

Reply:
I like `EmptyOnly`. Another option could be `UnsetOnly`, which are more specific to namespace scoped resources. This resource are set to "default" in deployment.

Reply:
👍 I like `UnsetOnly`.